types/unl-core/index.d.ts

```diff
@@ -24,20 +24,21 @@ export interface Point {
     lon: number;
 }
 
+export interface PointWithElevation extends Point {
```

Comment:
Has this changed? If yes then why PointWithElevation. extends Point and still defines lat an lon?